scripts/merge_repos_and_check_duplicates.py

```
@@ -1,97 +1,118 @@
-    
-    duplicated_files = find_duplicates(files)    
-    
-    # check if duplicates exist
+
+    duplicated_files = find_duplicates(files)
+
     if len(duplicated_files) > 0:
-        sys.stdout.write(f'Skip start: duplicate files were found while copying files to a {general_folder} folder: '+ ',\n'.join(duplicated_files) + '\n')
+        sys.stdout.write(
+            "Skip start: duplicate files were found while copying files to a "
+            f"{general_folder} folder: " + ",\n".join(duplicated_files) + "\n"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)